src/App.tsx

```diff
@@ -121,18 +111,18 @@ const App: React.FC = () => {
         {user.isLoggedIn && <AdminMenu />}
         <Breadcrumbs />
         {showAuthorityOrcidModal && <AuthorityOrcidModal />}
-        <StyledPageBody>
+        <ContentPage>
           <Switch>
             <Route exact path="/" component={Dashboard} />
-            {user.isLoggedIn && <Route exact path="/publications" component={Workspace} />}
-            {user.isLoggedIn && <Route exact path="/publications/new" component={NewPublication} />}
+            {user.isLoggedIn && <Route exact path="/new-publication" component={NewPublication} />}
```

Comment:
Når vi får på plass fetch resource, så skal samme visning brukes for edit også, så tror vi bare bør bruke `/publications/{pub_id}`.
Men det kan vi ta når enpoint er klart også

Reply:
samme tenkte jeg